src/System.Device.Gpio.Tests/README.md

```diff
@@ -0,0 +1,37 @@
+# How to run tests on the Raspberry Pi
+
+Prerequisites: 
+- Installed .NET SDK with the "dotnet" executable in the path
+- Locally cloned repository
+
+Build System.Device.Gpio.dll:
+```
+cd src/System.Device.Gpio
+dotnet build System.Device.Gpio.sln
+```
+This builds the main System.Device.Gpio assembly together with its test assembly. Before running the tests, you need to:
+- Connect BCM Pins 12 and 16 (physical Pins 32 and 36) with a 10kΩ resistor between them. 
```

Comment:
Technically speaking no resistor is required. Resistor is added so that it's not possible to short-circuit. Any resistor should be fine but I'd recommend anything higher than 1k and not too high so that internal pull up/down doesn't win

Reply:
What should I add? @joperezr above suggested to include the Resistor, I originally skipped it (which worked as well). Should I write something like: 
Connect BCM Pins 12 and 16 (physical Pins 32 and 36) with a cable. To protect from shortening Pins, a resistor between 1kΩ and 10kΩ should be added in the line.

Reply:
Note: See also https://raspberrypi.stackexchange.com/questions/104462/does-directly-connecting-pins-fry-the-pi which gives a good answer on what could happen.